generated/nidcpower/nidcpower/lcr_measurement.py

```diff
@@ -0,0 +1,210 @@
+        ("vdc"                 , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("idc"                 , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("stimulus_frequency"  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_voltage_real"     , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_voltage_imaginary", nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_current_real"     , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ac_current_imaginary", nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_real"              , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_imaginary"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_magnitude"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("z_phase"             , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_real"              , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_imaginary"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_magnitude"         , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("y_phase"             , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("ls"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("cs"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("rs"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("lp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("cp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("rp"                  , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("d"                   , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("q"                   , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("measurement_mode"    , nidcpower._visatype.ViUInt16 ),  # noqa: E202,E203
+        ("dc_in_compliance"    , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("ac_in_compliance"    , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("unbalanced"          , nidcpower._visatype.ViBoolean),  # noqa: E202,E203
+        ("reserved1"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved2"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved3"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved4"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved5"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved6"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+        ("reserved7"           , nidcpower._visatype.ViReal64 ),  # noqa: E202,E203
+    ]
+
+
+class LCRMeasurement(object):
+    """Specifies an LCR measurement.
+
+    Data attributes:
```

Comment:
Not all the supported properties are documented here.
It looks like only those that are implemented by setting `_self.foo` are.
But that's an implementation detail as far as I understand and irrelevant to the user.

Reply:
Tracked by #1781.